### DISCUSSION TITLE
To ask about change the expression of transporters in PKSim

### DISCUSSION THREAD

**Discussion by dat994** [Open-Systems-Pharmacology/Forum#1857] [Questions & Problems]
Hello everyone, 
I need to change the expression of a transporter for simulation. I went to the expression profile tab to see the expression profile of my targeted protein. When I see the expression profile I find it quite difficult to understand. 
First is all the proteins, whether enzymes or transporters, the relative expression is set to 0 at the default, it means the model doesn't use the protein even if we set the model as the substrate of enzyme and transporter. I think I am wrong at this point but I can't explain at this point. 
Secondly, my interested protein is ABCG2, I want to increase the expression of this protein 2 folds in the kidney, duodenum and jejunum for example,  however since the relative expression is set to 0 at default, how can I change the expression of this protein? Can you suggest me this step?
Thank you in advance, 
Dat

**Comment by HenrikCordes** [comment]
Hi @dat994,

the OSP-Suite specific concept of using gene expression data is nicely outlined in the help: 
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile 
and in primary literature: 
https://pubmed.ncbi.nlm.nih.gov/22293118/ 
https://onlinelibrary.wiley.com/doi/10.1002/psp4.12904

The concept builds up on two distanced information:
1. The reference concentration (how much protein in the compartment associated to the highest expression value e.g. 100%)
2. The expression pattern (relative amount compared the compartment with the highest expression)

Best, 
Henrik
